examples/basic/lines/line_single.py

```diff
@@ -1,3 +1,15 @@
+'''A line graph using user-defined data points. This example demonstrates
+   the use of the ``line`` function to make a graph by mapping straight lines
+   between defined points.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line
+    :refs: :ref:`ug_basic_lines_single`
+    :keywords: line, glyph, straight
```

Comment:
We usually stick to nouns/concepts for keywords 
```suggestion
    :keywords: line, glyph
```